### PR TITLE
pango: update to 1.50.2

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by ericonr** [comment]
I've found the musl test failure. Our `glib` is misconfigured due to a lack of `CFLAGS` (upstream's fault, I'm not sure how it went undetected before).

It should be using the `USE_XLOCALE` code path in `gstrfuncs.c`, but it isn't, so it's using the `g_ascii_formatd` implementation, which has a mistaken `  g_return_val_if_fail (format[0] == '%', NULL);` for some reason. It fails, because the format being passed is `" %f"` (note the space).

I'm not sure why the test exists, but trying to fix glib first.

**Comment by ericonr** [comment]
My mistake, glib isn't misconfigured, musl doesn't have `strtoll_l` or `strtoull_l`.

So possibly patching out the assert is enough, will check.

**Comment by ericonr** [comment]
the code is 20y old, I'm curious how it was never caught before :P

**Comment by ericonr** [comment]
Am trying again.

File: srcpkgs/pango/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'pango'
 pkgname=pango
-version=1.48.10
+version=1.50.0
```
Comment (ericonr) [inline srcpkgs/pango/template:3]: 1.50.1 is out.

**Comment by cinerea0** [comment]
Temporarily marking as draft so I can test 1.50.2 fully.

**Comment by cinerea0** [comment]
@ericonr The checks are failing again.

**Comment by ericonr** [comment]
Oof, these are different failures...

**Comment by ericonr** [comment]
So uh, tests work here :p

**Comment by ram02z** [comment]
This pango update broke text rendering on swaybar.

**Comment by ericonr** [comment]
@ram02z can you please open an issue and share screenshots?

I don't think this has been reported to sway upstream yet :/

**Comment by cinerea0** [comment]
@ram02z Also, could you check if the update in #34725 fixes your problem?

**Comment by ram02z** [comment]
> @ram02z can you please open an issue and share screenshots?
> 
> I don't think this has been reported to sway upstream yet :/

Sorry, it was i3status-rs, not swaybar. 
Seems to be this https://github.com/greshake/i3status-rust/issues/1367 

> @ram02z Also, could you check if the update in #34725 fixes your problem?

Yeah, pango 1.50.3 fixes the bug.

**Pull request merged**